CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 2.1.4 – 2022-11-30
+### Fixed
+- Show only existing mailboxes, not all subscribed mailboxes
```

Comment:
it's the other way around

we now show all mailboxes again, including virtual ones